deploy/aws.md

```diff
@@ -127,6 +126,12 @@ The default Cloudfront URL for your tiles:
 https://SUBDOMAIN.cloudfront.net/ARCHIVE_NAME/{z}/{x}/{y}.{ext}
 ```
 
+If you're using [MapLibre](/pmtiles/maplibre), it's more convenient to fetch [TileJSON](https://github.com/mapbox/tilejson-spec/tree/master/3.0.0), which will detect the tileset `minzoom` and `maxzoom`:
```

Comment:
Nit: This new section might better be placed _below_ the next pre-existing paragraph that starts `where {ext} is the file extension...` as that explains the symbols in the example above and now they are separated in this PR?

Reply:
Good catch, fixed!